tutorials/llama_index_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<p>might link to documentation as to how you wold do this via llama_index docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>👍</p>